cards/kingdom_cards/dominion.py

```diff
@@ -196,15 +196,19 @@ class Militia(Card):
     adhocturn_form = MilitiaForm
 
     def perform_specific_action(self, deck, turn):
+        result = []
```

Comment:
Minor nit-pick: `result` signifies a single thing. `results` is a better name (since it's multiple objects being returned.